src/concurrency.md

```diff
@@ -3,6 +3,7 @@
 | Recipe | Crates | Categories |
 |--------|--------|------------|
 | [Mutate the elements of an array in parallel][ex-rayon-iter-mut] | [![rayon-badge]][rayon] | [![cat-concurrency-badge]][cat-concurrency] |
+| [Check the elements of an array against a predicate in parallel][ex-rayon-any-all] | [![rayon-badge]][rayon] | [![cat-concurrency-badge]][cat-concurrency] |
```

Comment:
How about renaming the example to simply:
"Test in parallel if all or any collection elements match given predicate"

We are not limited to arrays here (and actually any collection/iterator viable for `par_Iter` will do)

Reply:
Definitely 👍